pkgs/applications/backup/timeshift/default.nix

```
@@ -0,0 +1,34 @@
+{ callPackage
+, lib
+, timeshift-wrapper ? (callPackage ./wrapper.nix { })
```

```suggestion
, timeshift-wrapper
```
Please move that to top-level.